Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,28 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
```

Comment:
Remember you have to calculate BMI for another person :)

Reply:
okay, I added this! but I am having a hard time understanding how to resolve the division issue.